misc/_posts/project-week-data-exercises.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: Project Week Data Exercises
+title: Preparatory Data Exercises for Project Week
+description:  # Add post description (optional)
+img:  # Add image post (optional)
+hidden: true
+tags: []
+---
+
+## "Over-the-break" Data Exercise Parts 1 & 2
+
+[Parts 1 & 2 (the "Over-the-break" Data Exercise):](https://docs.google.com/document/d/1CRU5y3CGDYsaPv1FBkQhJ8ESTtem70RHQ1Q3AmVdOv0/edit) The first two installations of the Project Week data exercises! Enrolled students should completed the assignment by copying and pasting the GitHub link to your code in the “Over-the-Break Data Exercise: Parts 1 & 2” Assignment on [Canvas](https://develop.fiu.edu/courses/abcd-repronim-course-reproducible-analyses-of-abcd-data). Observer Students should complete the assignment by copying and pasting the GitHub link to your code in this [Google Form](https://docs.google.com/forms/d/e/1FAIpQLSe537NZ0jD6pj52RctDIQPjOxiSQJQK53XNXI7YY-wZEAE7Cw/viewform?usp=sf_link).
```

Comment:
It's easier to review if you do one sentence per line, but I'll respond with free text instead of recommending changes:

"Enrolled students should completed" --> "Enrolled students should complete"

"Observer Students" --> "Observer students"

Reply:
Thanks will do this next time. And good catches.